Bottom: 0942a13dc4b99cbed5a974432f23a4c22618c1d8
Top:    29015faf3e5bf71601af9803e0a725c47164dc5a
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-04-25 13:55:55 -0500

Remove backup directory creation

Roles create their own backup directories


---
diff --git a/roles/docker-common/tasks/main.yml b/roles/docker-common/tasks/main.yml
index b588fb6..7f9e8f5 100644
--- a/roles/docker-common/tasks/main.yml
+++ b/roles/docker-common/tasks/main.yml
@@ -9,15 +9,6 @@
 - import_tasks: sshd_settings.yml
 - import_tasks: mariadb_settings.yml
 
-- name: restores base dir
-  file:
-    state: directory
-    path: '{{ docker_restore_dir }}'
-    owner: root
-    group: tape
-    mode: 'u=rwx,g=rwx,o=rx'
-    recurse: no
-
 - name: base docker_container.conf dir
   file:
     state: directory
@@ -35,22 +26,3 @@
 
 - include_tasks: selinux-backula-policy.yml
   when: (docker_bacula_pp.stat.exists is defined) and (docker_bacula_pp.stat.exists == false)
-
-# *****************************************************************************
-# backup script part
-#   Docker backup script parts should be between 50 and 59
-
-- name: Assemble dir for backup scripts
-  file:
-    path: /usr/libexec/bacula/backup-scripts
-    state: directory
-
-- name: Create before_backup script
-  template:
-    src: before_backup.j2
-    dest: /usr/libexec/bacula/backup-scripts/50.before_backup.docker
-
-- name: Create after_backup script
-  template:
-    src: after_backup.j2
-    dest: /usr/libexec/bacula/backup-scripts/59.after_backup.docker
